configs/config.yaml

```
@@ -49,4 +49,43 @@ firecrawl:
+      type: "betterstack"
+      enabled: true
+      options:
+        source_token: "${BETTERSTACK_SOURCE_TOKEN}"
+        endpoint: "${BETTERSTACK_ENDPOINT}"
+        batch_size: 100
+        flush_interval: "5s"
+        max_retries: 3
+        timeout: "30s"
+        user_agent: "letraz-utils/1.0"
+        headers:
+          Content-Type: "application/json" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)